app/src/main/java/app/revanced/integrations/music/patches/misc/AlternativeDomainPatch.java

```diff
@@ -0,0 +1,38 @@
+package app.revanced.integrations.music.patches.misc;
+
+import static app.revanced.integrations.music.settings.Settings.ALT_THUMBNAIL_ALTERNATIVE_DOMAIN;
+import static app.revanced.integrations.music.settings.Settings.ALT_THUMBNAIL_USE_ALTERNATIVE_DOMAIN;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import app.revanced.integrations.shared.utils.Logger;
+
+
+public final class AlternativeDomainPatch {
+
+    private static final String YOUTUBE_STATIC_THUMBNAILS_DOMAIN_REGEX = "(yt[3-4]|lh[3-6]|play-lh)\\.(ggpht|googleusercontent)\\.com";
```

Comment:
Match all thumbnails domain. Can u make this changes for YouTube also ?

```suggestion
    private static final String YOUTUBE_STATIC_THUMBNAILS_DOMAIN_REGEX = "(ap[1-2]|gm[1-4]|gz[0]|(cp|ci|gp|lh)[3-6]|sp[1-3]|yt[3-4]|(play|ccp)-lh)\\.(ggpht|googleusercontent)\\.com";
```

Reply:
Image servers for YouTube and YouTube Music may be different
Since this is a different issue from this PR, it should be handled in a separate PR